### ISSUE TITLE
v 2.2.18 of parse-server code crashes on deployment

### ISSUE THREAD

**Issue by sprabs** [ParsePlatform/parse-server#2666]
### Issue Description

I have been using parse-server for a few months now on my AWS Elastic Beanstalk and have never had a problem deploying my app. When I pulled v 2.2.18, I got these errors:

> [Instance: i-adca39b5] Command failed on instance. Return code: 1 Output: (TRUNCATED)..."/opt/elasticbeanstalk/containerfiles/ebnode.py", line 180, in npm_install raise e subprocess.CalledProcessError: Command '['/opt/elasticbeanstalk/node-install/node-v4.4.6-linux-x64/bin/npm', '--production', 'rebuild']' returned non-zero exit status 1. Hook /opt/elasticbeanstalk/hooks/appdeploy/pre/50npm.sh failed. For more detail, check /var/log/eb-activity.log using console or EB CLI.
> 
> Unsuccessful command execution on instance id(s) [REDACTED]. Aborting the operation.
> 
> Failed to deploy application.

Any idea what the issue can be? Here is what my package.json looks like in case anything is wrong here:

  ```
"name": "something",
  "version": "1.0.0",
  "description": "Parse Server Code",
  "main": "index.js",
  "repository": {
    "type": "git",
    "url": "git://github.com/OrgName/ProjectRepo.git"
  },
  "license": "MIT",
  "dependencies": {
    "express": "=4.14.0",
    "kerberos": "~0.0.x",
    "parse": "~1.6.12",
    "parse-server": "2.2.18",
    "twilio": "^2.9.2",
    "underscore": "*"
  },
  "scripts": {
    "start": "node index.js"
  },
  "engines": {
    "node": ">=4.3"
  },
  "devDependencies": {
    "babel-cli": "^6.6.4"
  }
```

### Environment Setup

- **Server**
  - parse-server version: 2.2.18
  - Operating System:     64bit Amazon Linux 2016.03 v2.1.3 running Node.js
  - Hardware:             AWS EC2 t2.medium
  - Localhost or remote server? AWS Elastic Beanstalk

In the past, I even had parse-server as a dev dependency and it worked fine. I tried doing that again here, but it didn't work. I run **npm install parse-server** prior to uploading and then package my Cloud Code, etc... accordingly to deploy.

**Comment by acinader** [comment]
@sprabs, have you ruled out running out of memory?

how much memory do you have allocated in your Dockerrun.aws.json?  Have you ssh'd to the container host and checked /var/log/messages?

**Comment by sprabs** [comment]
@acinader I just checked /var/log/messages. Nothing much there.

I have ruled out running out of memory. I've deployed maybe 20x in the last 24 hours with the previous version of parse-server without any problems. t2.medium is also a pretty solid machine for our dev setup (not too many RPS flowing through). Did something substantial change in the build from 2.2.17 to 2.2.18??? Not sure why this would happen. Or could it be in any way related to my dependencies in package.json?
